assets/js/actions/Dismiss.js

```diff
@@ -3,9 +3,10 @@
 const utils = require('../libs/elife-utils')();
 
 module.exports = class Dismiss {
-  constructor($parent, $patternRoot, document, cookieExpiry) {
+  constructor($parent, $toDismiss, document, cookieExpiry) {
+    this.$toDismiss = $toDismiss;
     this.doc = document;
-    this.cookieName = Dismiss.deriveCookieName($patternRoot);
+    this.cookieName = Dismiss.deriveCookieName(this.$toDismiss);
```

Comment:
To dismiss what? I do not think this clear as what we are dismissing.

Reply:
`$toDismiss` is the element to dismiss. We know it's an element because it's `$`-prefixed, according to the convention for this codebase. It's a more meaningful name than `$patternRoot`, as it tells you why it's there, rather than just what it is.